templates/zerver/help/configure-email-notifications.md

```diff
@@ -0,0 +1,17 @@
+# Configure email notifications
+
+Zulip allows you to configure your email notification preferences. You have a choice to receive or not receive email notification when you're offline.
+
+You can easily change your email notification settings by following these steps.
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right   corner of the right sidebar.
+
+2. Select **Settings** from the dropdown menu that appears.<br>
+    ![Settings](/static/images/help/cog_dropdown_settings.png)
+
+3. Scroll down to the **Notifications** section (after the **Display Settings** section) to view the message display options.
+
+4. Under the **Other notifications I want** section of **Notifications** you should find a check box with the lable **Digest emails when I'm away**. When this checkbox is selected you will receive email notifications whenever you're offline.
```

Comment:
> When this checkbox is selected you will receive email notifications whenever you're offline.

Digest emails are not same as email notifications that you receive when you are offline. 

The option for enabling email notifications when you are offline is under **Private messages and @-mentions**